_data/changelogs/docs-settings.yml

```
@@ -2,6 +2,17 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: 2023-05-12
```

I noticed in the [3.5.0 changelog PR](https://github.com/uswds/uswds-site/pull/2096/files) the standard is to start adding the date as `NNNN-NN-NN`, presumably for bulk replacement. Thinking we should probably start making that a standard practice. 
```suggestion
  - date: NNNN-NN-NN
```

---

Resolved in [e6b26d8](https://github.com/uswds/uswds-site/pull/2077/commits/e6b26d84f667f9e2ab669bd51dbb70d345bc6b64)